src/pages/archive/archive.jsx

```diff
@@ -1,13 +1,26 @@
 import "../home/home.css";
-import { Navbar, NoteContainer, Sidebar } from "components";
+import { NoteContainer, Sidebar } from "components";
+import { useNote } from "context";
 
 export const Archive = () => {
+  const { noteArrayState } = useNote();
   return (
     <>
-      <Navbar />
       <section className="home">
         <Sidebar />
-        <NoteContainer />
+        <main className="content">
+          {noteArrayState.archives.map((archive, index) => (
```

Comment:
you can destructure archive so that you have to not write ```archive.``` everytime

Reply:
done